selfdrive/ui/ui.h

```diff
@@ -47,40 +47,40 @@ struct Alert {
     return text1 == a2.text1 && text2 == a2.text2 && type == a2.type && sound == a2.sound;
   }
 
-  static Alert get(const SubMaster &sm, uint64_t started_frame) {
+  static std::optional<Alert> get(const SubMaster &sm, uint64_t started_frame) {
```

Comment:
I think Alert::get should always return an alert to show, so the std::optional doesn't make sense. instead, i'd add some logic to handle this case inside of Alert::get